test/custom-task-ctrls/wait-task-beta/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.15
 require (
 	github.com/emicklei/go-restful v2.15.0+incompatible // indirect
 	github.com/google/go-cmp v0.5.9
-	github.com/tektoncd/pipeline v0.42.0
+	github.com/tektoncd/pipeline v0.43.2
```

Comment:
Wondering if this shall be put in the release note?

Reply:
This is a controller used interally for integration test, it's not user facing :)